AutoMapper.AspNetCore.OData.EFCore/QueryableExtensions.cs

```diff
@@ -132,6 +132,10 @@ private static IQueryable<TModel> GetQueryable<TModel, TData>(this IQueryable<TD
         {
             
             var expansions = options.SelectExpand.GetExpansions(typeof(TModel));
+            
+            var selects = options.SelectExpand.GetSelects();
```

Comment:
Wanted to add this logic to `.BuildIncludes` itself, but I saw it resides in a different repo (LogicBuilder) & got hard time with it